docs/ref/models/expressions.txt

```diff
@@ -677,6 +862,30 @@ calling the appropriate methods on the wrapped expression.
         Tells Django that this expression contains an aggregate and that a
         ``GROUP BY`` clause needs to be added to the query.
 
+    .. attribute:: contains_over_clause
+
+        .. versionadded:: 2.0
+
+        Tells Django that this expression contains a
+        :class:`~django.db.models.expressions.Window` expression. Queries with a
+        window function expression are mainly disallowed in queries that
```

Comment:
You can't use them in `INSERT` or `UPDATE` (at least not in Django, I think a work-around in SQL is common table expressions although I've never tried), and PostgreSQL has a caveat with `FOR UPDATE/SHARE`. I used "mainly" because there may be other queries where an error is raised by the database. I think I'm more keen to delete "usually", and try to replace with something like "For instance, window function expressions can't be used `INSERT` or `UPDATE` queries".